src/main/kotlin/app/revanced/patches/youtube/misc/integrations/fingerprints/EmbeddedPlayerControlsOverlayFingerprint.kt

```diff
@@ -4,12 +4,19 @@ import app.revanced.patcher.extensions.or
 import app.revanced.patches.shared.integrations.patch.AbstractIntegrationsPatch.IntegrationsFingerprint
 import org.jf.dexlib2.AccessFlags
 
+/**
+ * For embedded playback inside Google Play store (and probably other situations as well).
+ *
+ * Note: this fingerprint may no longer be needed, as it appears
+ * [RemoteEmbedFragmentFingerprint] may be set before this hook is called.
+ */
 object EmbeddedPlayerControlsOverlayFingerprint : IntegrationsFingerprint(
     accessFlags = AccessFlags.PRIVATE or AccessFlags.CONSTRUCTOR,
     returnType = "V",
-    parameters = listOf("L", "L", "L"),
+    parameters = listOf("Landroid/content/Context;", "L", "L"),
```

Comment:
Only the first character is checked. If you think it adds semantic context though, you can keep it.
```suggestion
    parameters = listOf("L", "L", "L"),
```

Reply:
Looks good.